src/contexts/LoginContext.tsx

```diff
@@ -33,45 +34,29 @@ const useLoginContext = (): LoginContextProps => {
 const LoginProvider = ({
   children,
 }: PropsWithChildren<Record<string, never>>): JSX.Element => {
-  const [storedUserId, setStoredUserId, removeStoredUserId] = useLocalStorage(
-    LOCAL_STORAGE_KEYS.GithubId,
-    "Unknown user"
-  )
-  const [
-    storedUser,
-    setStoredUser,
-    removeStoredUser,
-  ] = useLocalStorage(LOCAL_STORAGE_KEYS.User, { email: "", contactNumber: "" })
-  const [
-    storedUserEmail,
-    setStoredUserEmail,
-    removeStoredUserEmail,
-  ] = useLocalStorage(LOCAL_STORAGE_KEYS.Email, "Unknown email")
-
   const [, , removeSites] = useLocalStorage(
     LOCAL_STORAGE_KEYS.SitesIsPrivate,
     false
   )
+  const [storedUserId, setStoredUserId] = useState("")
+  const [storedUserContact, setStoredUserContact] = useState("")
+  const [storedUserEmail, setStoredUserEmail] = useState("")
   const verifyLoginAndSetLocalStorage = useCallback(async () => {
     const { data: loggedInUser } = await axios.get<LoggedInUser>(
```

Comment:
since we removed the local storage keys, for example in the phone verification flow where the number was pre-filled, will it re-fetch user details from API ?

Reply:
Yeah it refetches - the number will still be prefilled for the user